<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1248</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vizualizador 3D</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="FrameDesenho" name="frameDesenho">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>400</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>20</y>
      <width>231</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tabTranslacao">
     <attribute name="title">
      <string>T</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>201</width>
        <height>136</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lblTranslacao">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Translação</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QLabel" name="lblTranslacaoZ">
           <property name="text">
            <string>z</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="spinTranslacaoY">
           <property name="minimum">
            <double>-100.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="lblTranslcaoY">
           <property name="text">
            <string>y</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QDoubleSpinBox" name="spinTranslacaoX">
           <property name="minimum">
            <double>-100.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lblTranslacaoX">
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QDoubleSpinBox" name="spinTranslacaoZ"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabEscala">
     <attribute name="title">
      <string>S</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>47</y>
        <width>201</width>
        <height>110</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="0">
        <widget class="QLabel" name="lblEscalaZ">
         <property name="text">
          <string>z</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblEscalaX">
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="spinEscalaY">
         <property name="minimum">
          <double>-100.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QDoubleSpinBox" name="spinEscalaX">
         <property name="minimum">
          <double>-100.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="lblEscalaY">
         <property name="text">
          <string>y</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QDoubleSpinBox" name="spinEscalaZ"/>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="lblEscala">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>199</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Escala</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabRotacao">
     <attribute name="title">
      <string>R</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>211</width>
        <height>290</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="4" column="1">
        <widget class="QSlider" name="hsRotacaoZ">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>-180</number>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblRotacaoZ">
         <property name="text">
          <string>Z</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="hsRotacaoX">
         <property name="minimum">
          <number>-180</number>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblRotacaoY">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSlider" name="hsRotacaoY">
         <property name="minimum">
          <number>-180</number>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblRotacaoX">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lblRotacao">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Rotação</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>131</width>
      <height>681</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblForma">
       <property name="text">
        <string>Forma:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboFormas"/>
     </item>
     <item>
      <widget class="QLabel" name="lblCoordenadaX1">
       <property name="text">
        <string>Coordenada X1:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinX1">
       <property name="maximum">
        <double>600.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCoordenadaY1">
       <property name="text">
        <string>Coodernada Y1:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinY1">
       <property name="maximum">
        <double>600.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCoordenadaZ1">
       <property name="text">
        <string>Coordenada Z1:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinZ1"/>
     </item>
     <item>
      <widget class="QLabel" name="lblCoordenadaX2">
       <property name="text">
        <string>Coordenada X2:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinX2">
       <property name="maximum">
        <double>600.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCoordenadaY2">
       <property name="text">
        <string>Coodernada Y2:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinY2">
       <property name="maximum">
        <double>600.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCoordenadaZ2">
       <property name="text">
        <string>Coordenada Z2:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinZ2"/>
     </item>
     <item>
      <widget class="QLabel" name="lblCoordenadaX3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Coordenada X3:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinX3">
       <property name="maximum">
        <double>600.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCoordenadaY3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Coordenada Y3:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinY3">
       <property name="maximum">
        <double>600.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCoordenadaZ3">
       <property name="text">
        <string>Coordenada Z3:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinZ3"/>
     </item>
     <item>
      <widget class="QLabel" name="lblRaio">
       <property name="text">
        <string>Raio:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinRaio">
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblTamanho">
       <property name="text">
        <string>Tamanho</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinTamanho"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnCor">
       <property name="text">
        <string>Selecionar Cor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDesenhar">
       <property name="text">
        <string>Adicionar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnModificarForma">
       <property name="text">
        <string>Modificar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExcluirForma">
       <property name="text">
        <string>Excluir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnCarregarOBJ">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>640</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Carregar</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbDisplayFile">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>640</y>
      <width>481</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbDFCameras">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>640</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimparSelecao">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>640</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpar Seleção</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxNormal">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>480</y>
      <width>89</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Vetor Normal</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="lblNx">
       <property name="text">
        <string>Nx</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinNx"/>
     </item>
     <item>
      <widget class="QLabel" name="lblNy">
       <property name="text">
        <string>Ny</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinNy"/>
     </item>
     <item>
      <widget class="QLabel" name="lblNz">
       <property name="text">
        <string>Nz</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinNz"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>410</y>
      <width>163</width>
      <height>56</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QRadioButton" name="rbTransformarObjeto">
       <property name="text">
        <string>Transformar objeto</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbTransformarCamera">
       <property name="text">
        <string>Transformar camera</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrameDesenho</class>
   <extends>QFrame</extends>
   <header>frame_desenho</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
